modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -234,24 +234,7 @@ protected Response uploadNoTransaction(@Context HttpServletRequest request,
         long uploadedSize = getUploadedSize(request);
         boolean isMultipart = contentType != null && contentType.contains("multipart");
```

Comment:
these are still used. So I keep them.

Reply:
I think it can be removed, and just give `false` to the `buildResponse` method, as there is no more multipart?